### ISSUE TITLE
`--js-dom` / `using jsDom` doesn't seem to work?

### ISSUE THREAD

**Issue by armanbilge** [VirtusLab/scala-cli#709]
**Version(s)**

0.1.2

**Describe the bug**

I added this to a project but the tests fail with this error, suggesting it's not actually working.
```
JavaScriptException: ReferenceError: document is not defined
```

I looked at the scala-cli sources and couldn't tell if this option has actually been implemented?

**To Reproduce**

If this _has_ been implemented then I'll follow-up with a reproducer.

**Expected behaviour**
Tests should run with the JSDOM JSEnv :) or, this option should not exist.

**Comment by lwronski** [comment]
@armanbilge 
Thanks for reporting.

IMO it's a bug and Scala CLI should allows to run test with JS DOM. It should be fixed in #796

**Comment by armanbilge** [comment]
@lwronski thank you for all your work on my issue reports, it's much appreciated!
